vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/split_test_frequency_cap.cc

```diff
@@ -0,0 +1,71 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/frequency_capping/exclusion_rules/split_test_frequency_cap.h"
+
+#include "base/metrics/field_trial.h"
+#include "base/optional.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ads/internal/bundle/creative_ad_info.h"
+
+namespace ads {
+
+namespace {
+
+const char kStudyName[] = "AdvertiserSplitTestStudy";
+
+base::Optional<std::string> GetSplitTestGroup(const std::string study_name) {
+  base::FieldTrial* field_trial = base::FieldTrialList::Find(study_name);
+  if (!field_trial) {
+    return base::nullopt;
+  }
+
+  return field_trial->group_name();
+}
+
+}  // namespace
+
+SplitTestFrequencyCap::SplitTestFrequencyCap() = default;
+
+SplitTestFrequencyCap::~SplitTestFrequencyCap() = default;
+
+bool SplitTestFrequencyCap::ShouldExclude(const CreativeAdInfo& ad) {
+  if (!DoesRespectCap(ad)) {
+    last_message_ = base::StringPrintf(
```

Comment:
does this get logged to disk in any circumstances? we probably don't want to log creative set IDs if they reveal user's browsing interests.

Reply:
@diracdeltas this is logged to console/disk only if the user enabled logging for rewards via cli args